hw/mips/malta.c

```diff
@@ -1262,6 +1288,10 @@ void mips_malta_init(MachineState *machine)
     /* create CPU */
     mips_create_cpu(machine, s, &cbus_irq, &i8259_irq);
 
+#ifdef CONFIG_TCG_LOG_INSTR
+  qemu_log_instr_init(first_cpu);
```

Comment:
It seems like this shouldn't be part of this commit?

Reply:
Ah, this crept in because logging was temporarily broken. If logging is now fixed this can probably go away.

Reply:
Removed it.